packages/examples/packages/send-flow/package.json

```diff
@@ -44,7 +44,9 @@
   },
   "dependencies": {
     "@metamask/rpc-errors": "^7.0.2",
-    "@metamask/snaps-sdk": "workspace:^"
+    "@metamask/snaps-sdk": "workspace:^",
+    "@metamask/superstruct": "^3.1.0",
+    "@metamask/utils": "^9.0.0"
```

Comment:
We use `11.2.0` in every other Snap

Reply:
Linting is gonna fail unless we do that here https://github.com/MetaMask/snaps/actions/runs/14087378330/job/39455042024